Back-End/controllers/admin_controllers/instructors_controllers.js

```diff
@@ -1,6 +1,15 @@
 const db = require('../../db');
 
+const instructors_details = (req, res) => {
+    const query = `SELECT * FROM users WHERE role_id = 2`;
+    db.query(query, (err, result) => {
+        if (err) throw err
+        res.json(result)
+    })
+}
+
 module.exports = {
+    instructors_details
     
```

Comment:
Add a trailing comma, also remove the empty line

Reply:
Noted